### DISCUSSION TITLE
docs/translation/chaos-engineering-observability-visual-metaphors/

### DISCUSSION THREAD

**Discussion by giscus** [rootsongjc/cloud-native-library#13] [Announcements]
# docs/translation/chaos-engineering-observability-visual-metaphors/

本文为可观察性引入了一个新的角色：视觉隐喻，并进行了一项研究，对比了视觉隐喻与传统表示的结果。

https://jimmysong.io/docs/translation/chaos-engineering-observability-visual-metaphors/

**Comment by rootsongjc** [comment]
通过视觉隐喻的方式来表示可观察性，这对于视觉设计者有很高的要求，可能受到文化和个人理解的影响，无法将信息准确的传达给观察者，可以将这种方式与传统图表方法结合展示。
